docusaurus/docs/recipes/video-player.md

```diff
@@ -0,0 +1,99 @@
+---
+id: video-player
+title: Adding a video player
+sidebar_label: Video player
+---
+
+It's often normal to have the necessity of having videos in your website. In this recipe we will show you how to use [`react-player`](https://github.com/CookPete/react-player) and some common use cases.
+
+## Walk-through
+
+### 1. Install `react-player`
+
+```sh
```

Comment:
Not entirely sure this is necessary, but i would add a `--save` arg here.

Reply:
I did check that and it's not necessary. It adds `react-player` to the dependencies with or without the `--save`.